INFO - 2024-09-20 14:14:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-09-20 15:39:02 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`auditoria_capas`.`users`, CONSTRAINT `FK_users_area` FOREIGN KEY (`fk_area`) REFERENCES `area` (`id_area`)) in C:\laragon\www\framework_ssii\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\laragon\www\framework_ssii\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\laragon\www\framework_ssii\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\laragon\www\framework_ssii\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\laragon\www\framework_ssii\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\laragon\www\framework_ssii\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\framework_ssii\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\framework_ssii\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\framework_ssii\app\Models\AdminModel.php(27): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\framework_ssii\app\Controllers\AdminController.php(50): App\Models\AdminModel->insertUser(Array)
#9 C:\laragon\www\framework_ssii\system\CodeIgniter.php(933): App\Controllers\AdminController->insertData()
#10 C:\laragon\www\framework_ssii\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
#11 C:\laragon\www\framework_ssii\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\framework_ssii\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\framework_ssii\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\framework_ssii\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-09-20 15:39:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`auditoria_capas`.`users`, CONSTRAINT `FK_users_area` FOREIGN KEY (`fk_area`) REFERENCES `area` (`id_area`))
[Method: POST, Route: admin/insertData]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`, `name`, `firstName`, `lastName`, `password`, `fk_area`, `created_at`, `updated_at`) VALUES (:email:, :name:, :firstName:, :lastName:, :password:, :fk_area:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Models\AdminModel.php(27): CodeIgniter\Model->insert([...])
 6 APPPATH\Controllers\AdminController.php(50): App\Models\AdminModel->insertUser([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->insertData()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-20 15:39:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`auditoria_capas`.`users`, CONSTRAINT `FK_users_area` FOREIGN KEY (`fk_area`) REFERENCES `area` (`id_area`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`email`, `name`, `firstName`, `lastName`, `password`, `fk_area`, `created_at`, `updated_at`) VALUES (\'\', \'\', \'\', \'\', \'$2y$10$MZ0Ps8kAwzu5kvDwbQhvs.kaExQ.JaaXzpCJ3d6zd6b9LE3lS/rba\', \'Open this select menu\', \'2024-09-20 15:39:02\', \'2024-09-20 15:39:02\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`email`, `name`, `firstName`, `lastName`, `password`, `fk_area`, `created_at`, `updated_at`) VALUES (\'\', \'\', \'\', \'\', \'$2y$10$MZ0Ps8kAwzu5kvDwbQhvs.kaExQ.JaaXzpCJ3d6zd6b9LE3lS/rba\', \'Open this select menu\', \'2024-09-20 15:39:02\', \'2024-09-20 15:39:02\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`, `name`, `firstName`, `lastName`, `password`, `fk_area`, `created_at`, `updated_at`) VALUES (:email:, :name:, :firstName:, :lastName:, :password:, :fk_area:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Models\AdminModel.php(27): CodeIgniter\Model->insert([...])
 8 APPPATH\Controllers\AdminController.php(50): App\Models\AdminModel->insertUser([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->insertData()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-20 15:39:02 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`auditoria_capas`.`users`, CONSTRAINT `FK_users_area` FOREIGN KEY (`fk_area`) REFERENCES `area` (`id_area`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `users` (`email`, `name`, `firstName`, `lastName`, `password`, `fk_area`, `created_at`, `updated_at`) VALUES (\'\', \'\', \'\', \'\', \'$2y$10$MZ0Ps8kAwzu5kvDwbQhvs.kaExQ.JaaXzpCJ3d6zd6b9LE3lS/rba\', \'Open this select menu\', \'2024-09-20 15:39:02\', \'2024-09-20 15:39:02\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`email`, `name`, `firstName`, `lastName`, `password`, `fk_area`, `created_at`, `updated_at`) VALUES (\'\', \'\', \'\', \'\', \'$2y$10$MZ0Ps8kAwzu5kvDwbQhvs.kaExQ.JaaXzpCJ3d6zd6b9LE3lS/rba\', \'Open this select menu\', \'2024-09-20 15:39:02\', \'2024-09-20 15:39:02\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`email`, `name`, `firstName`, `lastName`, `password`, `fk_area`, `created_at`, `updated_at`) VALUES (\'\', \'\', \'\', \'\', \'$2y$10$MZ0Ps8kAwzu5kvDwbQhvs.kaExQ.JaaXzpCJ3d6zd6b9LE3lS/rba\', \'Open this select menu\', \'2024-09-20 15:39:02\', \'2024-09-20 15:39:02\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`email`, `name`, `firstName`, `lastName`, `password`, `fk_area`, `created_at`, `updated_at`) VALUES (:email:, :name:, :firstName:, :lastName:, :password:, :fk_area:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Models\AdminModel.php(27): CodeIgniter\Model->insert([...])
 9 APPPATH\Controllers\AdminController.php(50): App\Models\AdminModel->insertUser([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->insertData()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-09-20 15:50:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:50:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:50:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:50:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:50:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:51:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:55:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:55:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:56:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:57:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:57:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:57:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:58:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-09-20 15:59:07 --> TypeError: App\Controllers\Home::index(): Return value must be of type string, none returned
[Method: GET, Route: /]
in APPPATH\Controllers\Home.php on line 10.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-09-20 15:59:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 15:59:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 16:00:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 16:00:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 16:01:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-09-20 16:02:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-09-20 21:53:11 --> mysqli_sql_exception: Unknown column 'audir.id_audit' in 'field list' in C:\laragon\www\framework_ssii\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\laragon\www\framework_ssii\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT `audir`....', 0)
#1 C:\laragon\www\framework_ssii\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `audir`....')
#2 C:\laragon\www\framework_ssii\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `audir`....')
#3 C:\laragon\www\framework_ssii\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `audir`....', Array, false)
#4 C:\laragon\www\framework_ssii\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\framework_ssii\app\Models\AuditModel.php(36): CodeIgniter\Model->__call('get', Array)
#6 C:\laragon\www\framework_ssii\app\Controllers\AccionsController.php(15): App\Models\AuditModel->getDataOfAudits()
#7 C:\laragon\www\framework_ssii\system\CodeIgniter.php(933): App\Controllers\AccionsController->getAudits()
#8 C:\laragon\www\framework_ssii\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AccionsController))
#9 C:\laragon\www\framework_ssii\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\framework_ssii\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\framework_ssii\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\framework_ssii\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
ERROR - 2024-09-20 21:53:11 --> mysqli_sql_exception: Unknown column 'audir.id_audit' in 'field list' in C:\laragon\www\framework_ssii\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\laragon\www\framework_ssii\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT `audir`....', 0)
#1 C:\laragon\www\framework_ssii\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `audir`....')
#2 C:\laragon\www\framework_ssii\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `audir`....')
#3 C:\laragon\www\framework_ssii\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `audir`....', Array, false)
#4 C:\laragon\www\framework_ssii\system\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\framework_ssii\app\Models\AuditModel.php(36): CodeIgniter\Model->__call('get', Array)
#6 C:\laragon\www\framework_ssii\app\Controllers\AccionsController.php(15): App\Models\AuditModel->getDataOfAudits()
#7 C:\laragon\www\framework_ssii\system\CodeIgniter.php(933): App\Controllers\AccionsController->getAudits()
#8 C:\laragon\www\framework_ssii\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AccionsController))
#9 C:\laragon\www\framework_ssii\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\framework_ssii\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\laragon\www\framework_ssii\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\laragon\www\framework_ssii\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-09-20 21:53:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'audir.id_audit' in 'field list'
[Method: GET, Route: accions/getAudits]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = :audit.status:
AND `departament`.`status` = :departament.status:', [...], false)
 2 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Models\AuditModel.php(36): CodeIgniter\Model->__call('get', [])
 4 APPPATH\Controllers\AccionsController.php(15): App\Models\AuditModel->getDataOfAudits()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AccionsController->getAudits()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AccionsController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-20 21:53:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'audir.id_audit' in 'field list'
[Method: GET, Route: accions/getAudits]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = :audit.status:
AND `departament`.`status` = :departament.status:', [...], false)
 2 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Models\AuditModel.php(36): CodeIgniter\Model->__call('get', [])
 4 APPPATH\Controllers\AccionsController.php(15): App\Models\AuditModel->getDataOfAudits()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AccionsController->getAudits()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AccionsController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-20 21:53:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'audir.id_audit' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = 1
AND `departament`.`status` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = 1
AND `departament`.`status` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = :audit.status:
AND `departament`.`status` = :departament.status:', [...], false)
 4 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Models\AuditModel.php(36): CodeIgniter\Model->__call('get', [])
 6 APPPATH\Controllers\AccionsController.php(15): App\Models\AuditModel->getDataOfAudits()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AccionsController->getAudits()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AccionsController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-20 21:53:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'audir.id_audit' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = 1
AND `departament`.`status` = 1')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = 1
AND `departament`.`status` = 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = :audit.status:
AND `departament`.`status` = :departament.status:', [...], false)
 4 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 5 APPPATH\Models\AuditModel.php(36): CodeIgniter\Model->__call('get', [])
 6 APPPATH\Controllers\AccionsController.php(15): App\Models\AuditModel->getDataOfAudits()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AccionsController->getAudits()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AccionsController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-20 21:53:11 --> [Caused by] mysqli_sql_exception: Unknown column 'audir.id_audit' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = 1
AND `departament`.`status` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = 1
AND `departament`.`status` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = 1
AND `departament`.`status` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = :audit.status:
AND `departament`.`status` = :departament.status:', [...], false)
 5 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Models\AuditModel.php(36): CodeIgniter\Model->__call('get', [])
 7 APPPATH\Controllers\AccionsController.php(15): App\Models\AuditModel->getDataOfAudits()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AccionsController->getAudits()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AccionsController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-09-20 21:53:11 --> [Caused by] mysqli_sql_exception: Unknown column 'audir.id_audit' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = 1
AND `departament`.`status` = 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = 1
AND `departament`.`status` = 1')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = 1
AND `departament`.`status` = 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `audir`.`id_audit`, `audit`.`no_audit`, `audit`.`date`, `audit`.`auditor`, `audit`.`status`, `departament`.`departament`, `machinery`.`machinery`, `shift`.`shift`
FROM `audit`
JOIN `departament` ON `departament`.`id_departament` = `audit`.`fk_departament`
JOIN `machinery` ON `machinery`.`id_machinery` = `audit`.`fk_machinery`
JOIN `shift` ON `shift`.`id_shift` = `audit`.`fk_shift`
WHERE `audit`.`status` = :audit.status:
AND `departament`.`status` = :departament.status:', [...], false)
 5 SYSTEMPATH\Model.php(930): CodeIgniter\Database\BaseBuilder->get()
 6 APPPATH\Models\AuditModel.php(36): CodeIgniter\Model->__call('get', [])
 7 APPPATH\Controllers\AccionsController.php(15): App\Models\AuditModel->getDataOfAudits()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AccionsController->getAudits()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AccionsController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
